### DISCUSSION TITLE
List of tips

### DISCUSSION THREAD

**Discussion by cgiovanetti** [JAXtronomy/.github#1] [Ideas]
I've been compiling a document with some tips and observations about various JAX functions (i.e., reshaping arrays is slow, map_coordinates is slow, use `.block_until_ready()` for profiling, etc.).  It's not exactly code--is this appropriate to share/develop on this repo?  Is there somewhere it makes sense to put?

**Comment by renecotyfanboy** [comment]
Hi @cgiovanetti ! Maybe it could be a good page in the [jax-astro-guides](https://github.com/JAXtronomy/jax-astro-guide) repo
